manifests/g/gopass/gopass/1.15.5/gopass.gopass.installer.yaml

```diff
@@ -0,0 +1,17 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: gopass.gopass
+PackageVersion: 1.15.5
+MinimumOSVersion: 7.0.0.0
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: GnuPG.Gpg4win
```

Comment:
```suggestion
```

Reply:
This package installed and launched normally without the dependency. Does having the dependency enable additional features or functionality within the package?

Reply:
Yes, this dependency is needed for basic functionality. See https://github.com/microsoft/winget-pkgs/pull/106108#issuecomment-1542742626